eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -178,6 +275,42 @@ namespace eos
         return integrate<GSL::QAGS>(f, s_min, s_max);
     }
 
+    double
+    BToDLeptonNeutrino::differential_a_fb_leptonic(const double & s) const
+    {
```

Comment:
Cant you just simply write for the function body:

  return _imp->numerator_differential_a_fb_leptonic(s)/_imp->normalized_differential_decay_width(s);

???